docs/content/tutorial/custom-steps.md

```diff
@@ -0,0 +1,154 @@
+4. Review the configuration and click **Create**.
+5. You're now in your app configuration's **Basic Information** page. Click **Install App**, then **Install to _your-workspace-name_**, then **Allow** on the screen that follows.
+
+### Obtaining and storing your environment variables {#environment-variables}
+
+Before you'll be able to successfully run the app, you'll need to obtain and set some environment variables.
+
+1. On the **Install App** page, copy your **Bot User OAuth Token**. You will store this in your environment as `SLACK_BOT_TOKEN` (we'll get to that next).
+2. Navigate to **Basic Information** and in the **App-Level Tokens** section , click **Generate Token and Scopes**. Add the [`connections:write`](https://api.slack.com/scopes/connections:write) scope, name the token, and click **Generate**. (For more details, refer to [understanding OAuth scopes for bots](https://api.slack.com/tutorials/tracks/understanding-oauth-scopes-bot)). Copy this token. You will store this in your environment as `SLACK_APP_TOKEN`.
+3. Follow [these instructions](https://confluence.atlassian.com/adminjiraserver0909/configure-an-incoming-link-1251415519.html) to create an external app link and to generate its redirect URL (the base of which will be stored as your APP_BASE_URL variable below), client ID, and client secret.
+4. Run the following commands in your terminal to store your environment variables, client ID, and client secret.
+5. You'll also need to know your team ID (found by opening your Slack instance in a web browser and copying the value within the link that starts with the letter **T**) and your app ID (found under **Basic Information**).
+
+**For macOS**
+```bash
+export SLACK_BOT_TOKEN=<your-bot-token>
+export SLACK_APP_TOKEN=<your-app-token>
+export JIRA_CLIENT_ID=<client-id>
+export JIRA_CLIENT_SECRET=<client-secret>
+```
+
+**For Windows**
+```bash
+set SLACK_BOT_TOKEN=<your-bot-token>
+set SLACK_APP_TOKEN=<your-app-token>
+set JIRA_CLIENT_ID=<client-id>
+set JIRA_CLIENT_SECRET=<client-secret>
+```
+
+## Setting up and running your local project {#configure-project}
+
+Clone the starter template onto your machine by running the following command:
+
+```bash
+git clone https://github.com/slack-samples/bolt-python-jira-functions.git
+```
+
+Change into the new project directory:
+
+```bash
+cd bolt-python-jira-functions
+```
+
+Start your Python virtual environment:
+
+**For macOS**
+```bash
+python3 -m venv .venv
+source .venv/bin/activate
+```
+
+**For Windows**
+```bash
+py -m venv .venv
+.venv\Scripts\activate
+```
+
+Install the required dependencies:
+
+```bash
+pip install -r requirements.txt
+```
+
+Rename the `.example.env` file to `.env` and replace the values for each of the variables listed in the file:
```

Comment:
You could also add the SLACK_BOT_TOKEN and SLACK_APP_TOKEN in here and not have to add them above